include/k4ainternal/k4aplugin.h

```diff
@@ -23,13 +23,16 @@ extern "C" {
 #endif
 
 /**
- * Current Version of the k4A Depth Engine Plugin API
+ * Current Version of the Azure Kinect Depth Engine Plugin
  *
- * \remarks The MAJOR version must be updated to denote any breaking change
- * that would cause an older SDK to not be able to use this plugin.
+ * \remarks
+ * When the depth engine plugin is updated this version should be increased.
```

Comment:
Sure this be more specific that when the interface it updated that this should be changed.